x-pack/metricbeat/tests/system/config/metricbeat.yml.j2

```diff
@@ -0,0 +1,124 @@
+###################### Metricbeat Configuration Template #######################
```

Comment:
This file is copied from https://github.com/elastic/beats/blob/master/metricbeat/tests/system/config/metricbeat.yml.j2